doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1354 @@
+    Picolibc is used by default in Zephyr.
+  * The nRF Desktop application image configurations to use Picolibc (:kconfig:option:`CONFIG_PICOLIBC`) by default.
+    Using the minimal libc implementation (:kconfig:option:`CONFIG_MINIMAL_LIBC`) no longer decreases the memory footprint of the application image for most of the configurations.
+  * Enabled :ref:`nrf_desktop_usb_state_sof_synchronization` (:ref:`CONFIG_DESKTOP_USB_HID_REPORT_SENT_ON_SOF <config_desktop_app_options>` Kconfig option) by default on the nRF54H Series SoC (:kconfig:option:`CONFIG_SOC_SERIES_NRF54HX`).
+    The negative impact of USB polling jitter is more visible in case of USB High-Speed.
+  * The Fast Pair sysbuild configurations to align the application with the sysbuild Kconfig changes for controlling the Fast Pair provisioning process.
+    The Nordic device models intended for demonstration purposes are now supplied by default in the nRF Desktop Fast Pair configurations.
+  * The :ref:`nrf_desktop_dvfs` to no longer consume the :c:struct:`ble_peer_conn_params_event` event.
+    This allows to propagate the event to further listeners of the same or lower priority.
+    This prevents an issue where :ref:`nrf_desktop_ble_latency` is not informed about the connection parameter update (it might cause missing connection latency updates).
+  * The Low Latency Packet Mode (LLPM) dependency in the :ref:`nrf_desktop_ble_conn_params`.
+    The module relies on the :kconfig:option:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option.
+    This allows using the module also when the Bluetooth LE controller is not part of the main application.
+  * Enabled the :ref:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT <config_desktop_app_options>` Kconfig option for the nRF54H20 DK.
+    The option mitigates HID report rate drops during DFU image transfer through the nRF Desktop dongle.
+  * Explicitly enabled the :kconfig:option:`CONFIG_BT_CTLR_ASSERT_HANDLER` Kconfig option in IPC radio image configurations of the nRF54H20 DK.
+    This is done to use an assertion handler defined by the IPC radio image.
+  * Disabled the UDC DWC2 DMA support (:kconfig:option:`CONFIG_UDC_DWC2_DMA`) on the nRF54H20 DK.
+    The DMA support is experimental, and disabling the feature improves USB HID stability.
+    Since nRF Desktop uses only small HID reports (report size is smaller than 64 bytes), the DMA does not improve performance.
+  * The nRF Desktop device names to remove the``52`` infix, because the nRF Desktop application supports other SoC Series also.
+    This change breaks backwards compatibility.
+    Peripherals using firmware from |NCS| 3.0.0 (or newer) will not pair with dongles using firmware from an older |NCS| release and the other way around.
+    The HID configurator script has also been aligned to the new naming scheme.
+
+* Removed:
+
+  * An imply from the nRF Desktop Bluetooth connectivity Kconfig option (:ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>`).
```

Comment:
Please remove the:

```
  * An imply from the nRF Desktop Bluetooth connectivity Kconfig option (:ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>`).
    The imply enabled a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`) if MPSL was used for synchronization between the flash memory driver and the radio (:kconfig:option:`CONFIG_SOC_FLASH_NRF_RADIO_SYNC_MPSL`).
    The workaround for the MPSL flash synchronization issue (NCSDK-29354 in the :ref:`known_issues` page) is no longer needed, as the issue is now fixed.
```
The change was enabled back here: https://github.com/nrfconnect/sdk-nrf/pull/21868

Reply:
Removed this entry.